.github/workflows/ci_cd.yml

```diff
@@ -31,11 +31,15 @@ jobs:
         with:
           miniforge-variant: Mambaforge
           miniforge-version: latest
+          channels: conda-forge,robotology
+          channel-priority: true
 
       - name: Dependencies
         shell: bash -l {0}
         run: |
-          mamba install python=${{ matrix.python }} casadi pytest
+          mamba install python=${{ matrix.python }} casadi pytest liecasadi adam-robotics idyntree meshcat-python ffmpeg-python matplotlib
+          mamba install metis=5.1.0 # Reminder for https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4857#issuecomment-1699470569
```

Comment:
I think this is not necessary anymore after https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/527 .

Reply:
Thanks, fixed in https://github.com/ami-iit/hippopt/pull/9/commits/535ce016d1620e06bbc4eef52de66d5de7d9f990